src/Routes/Books/AddBook.jsx

```diff
@@ -45,7 +45,7 @@ const AddBook = () => {
 
   return (
     <div className="container-fluid mt-5">
-      <h4 className="text-muted">ADD NEW BOOK</h4>
+      <h4 className="text-muted fw-bold">ADD NEW BOOK</h4>
       <form onSubmit={handleSubmit}>
         <div className="row">
           <div className="col-md-4 my-2">
```

Comment:
- [ ] Kindly improve the styling of the form component. The input boxes and button heights should be reduced to match the design

- [ ] The button should also be longer with enough space on the left and right of the text.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![ywa](https://user-images.githubusercontent.com/54848363/222882178-a2696917-62ad-413c-9190-ce402e779a1d.jpg) | ![ioqa](https://user-images.githubusercontent.com/54848363/222882179-2346842e-f2f1-431d-baff-ec11a40ec1fb.jpg)

Reply:
Thanks for the information, I am changing the style to match with template